src/govuk/components/accordion/accordion.yaml

```
@@ -35,7 +35,7 @@ params:
     - name: summary.html
       type: string
       required: false
-      description: HTML content for summary line.
+      description: HTML content for summary line. You can only include [phrasing content](https://html.spec.whatwg.org/#phrasing-content) inside the summary line. This content should be simple. Do not use other elements besides phrasing content, as they will not be valid HTML. Any other elements are not likely to be an appropriate use of the accordion component.
```

Could we add something about how this restriction is due to the fact that the summary line is inside `<button>` element and buttons can only contain phrasing content.

---

@hannalaakso Good shout! How does this look?

```suggestion
      description: HTML content for summary line. The summary line is inside the button element, so it can only contain [phrasing content](https://html.spec.whatwg.org/#phrasing-content). This content should be simple. Do not use other elements besides phrasing content, as they will not be valid HTML. Any other elements are not likely to be an appropriate use of the accordion component.
```

---

Thanks @EoinShaughnessy, that looks good.

A couple more small suggestions:
> The summary line is inside the HTML `<button>` element

to clarify it's the HTML element we're talking about here, not a CSS class name.


> as they will not be valid HTML.

I would change this to say that the resulting HTML (ie. having non phrasing content inside a button) will not be valid (it's not that the non phrasing content elements by themselves are not valid).

> Any other elements are not likely to be an appropriate use of the accordion component. 

I'm unsure about this line (I just wrote it quickly to try and describe the issue). I would try to clarify it to something like

> The content should be simple. If you need to use non phrasing content inside the summary line, you should consider simplifying your content or reviewing the [guidance](https://design-system.service.gov.uk/components/accordion/#when-not-to-use-this-component) to understand whether the accordion is the correct component to use.

---

```suggestion
      description: HTML content for summary line. You can only include [phrasing content](https://html.spec.whatwg.org/#phrasing-content) inside the summary line. This content should be simple. Do not use other elements besides phrasing content, as they will not be valid HTML. Any other elements are not likely to be an appropriate use of the accordion component.
```

---

@hannalaakso Thanks! That sounds good.

> If you have considered using non-phrasing content inside the summary line, you should consider simplifying your content or reviewing the guidance to understand whether the accordion is the correct component to use.

Even if they need to, isn't it the case that they won't be able to (because the summary line can only contain phrasing content)?

```suggestion
      description: HTML content for summary line. The summary line is inside the HTML <button> element, so it can only contain [phrasing content](https://html.spec.whatwg.org/#phrasing-content). This content should be simple. Do not use other elements besides phrasing content, as the resulting HTML will not be valid. If you had planned to use non-phrasing content inside the summary line, you should instead:

- consider simplifying your content
- review the [accordion component's guidance](https://design-system.service.gov.uk/components/accordion/#when-not-to-use-this-component) to understand whether it's the right component to use

```